src/implementations/DopplerLegacyClientImpl.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  DopplerLegacyClient,
+  DopplerLegacyUserData,
+} from "../abstractions/doppler-legacy-client";
+import { ResultWithoutExpectedErrors } from "../abstractions/common/result-types";
+import { AxiosResponse, AxiosStatic } from "axios";
+import { AppConfiguration } from "../abstractions";
+
+export class DopplerLegacyClientImpl implements DopplerLegacyClient {
+  private instance;
+
+  constructor({
+    axiosStatic,
+    appConfiguration: { dopplerLegacyBaseUrl },
+  }: {
+    axiosStatic: AxiosStatic;
+    appConfiguration: Partial<AppConfiguration>;
+  }) {
+    this.instance = axiosStatic.create({
+      baseURL: dopplerLegacyBaseUrl,
+      withCredentials: true,
+    });
+  }
+
+  async getDopplerUserData(): Promise<
+    ResultWithoutExpectedErrors<DopplerLegacyUserData>
+  > {
+    try {
+      const axiosResponse: AxiosResponse<DopplerLegacyUserData> =
+        await this.instance.get("/WebApp/GetUserData");
+      const { jwtToken, unlayerUser } = axiosResponse.data;
+      return {
+        success: true,
+        value: {
+          jwtToken,
+          unlayerUser: {
+            id: unlayerUser.id,
+            signature: unlayerUser.signature,
+            email: unlayerUser.email,
```

Comment:
I have removed email field: https://github.com/FromDoppler/doppler-editors-webapp/pull/46

```suggestion
```

Reply:
See https://github.com/FromDoppler/doppler-editors-webapp/blob/22a29fc6343ded129084a662c02b099c030e479f/src/abstractions/doppler-legacy-client/index.ts#L3-L6

Reply:
Yep, I fixed that.